tests/providers/test_company.py

```diff
@@ -45,7 +45,7 @@ def test_company(self):
         company = self.factory.company()
         assert isinstance(company, six.string_types)
         assert any(prefix in company for prefix in prefixes)
-        assert any(company.startswith(prefix) for prefix in prefixes)
+        assert any(company.startswith(prefix) or company.endswith(prefix) for prefix in prefixes)
```

Comment:
can you also add a test that calls the new `company_category` method?

Reply:
@fcurella Added some tests 👍